README.md

```diff
@@ -14,72 +15,118 @@ OSPSuite.ReportingEngine implementation in R
 
-```R
-devtools::load_all()
+Once the bundles downloaded, you can install the packages by using the code below:
+
+```r
+# bundlePath <- path/to/osp/bundles
+install.packages(file.path(bundlePath,"ospsuite.utils.zip"), repos = NULL)
+install.packages(file.path(bundlePath,"tlf.zip"), repos = NULL)
+install.packages(file.path(bundlePath,"rSharp.zip"), repos = NULL)
+install.packages(file.path(bundlePath,"ospsuite.zip"), repos = NULL)
+```
+
+- Required packages from [CRAN](https://cran.r-project.org/):
+  > Some of these packages may already be installed as dependencies of the Open Systems Pharmacology ecosystem
+  - [`{dplyr}`](https://cran.r-project.org/web/packages/dplyr)
+  - [`{ggplot2}` >= 3.3.0](https://cran.r-project.org/web/packages/ggplot2) 
+  - [`{jsonlite}`](https://cran.r-project.org/web/packages/jsonlite)
+  - [`{R6}`](https://cran.r-project.org/web/packages/R6)
+  - [`{tidyr}`](https://cran.r-project.org/web/packages/tidyr)
+- Optional packages from [CRAN](https://cran.r-project.org/)
+  - [`{crayon}`](https://cran.r-project.org/web/packages/crayon)
+  - [`{knitr}`](https://cran.r-project.org/web/packages/knitr)
+  - [`{parallel}`](https://cran.r-project.org/web/packages/parallel)
+  - [`{readxl}`](https://cran.r-project.org/web/packages/readxl)
+  - [`{Rmpi}`](https://cran.r-project.org/web/packages/Rmpi)
+  - [`{styler}`](https://cran.r-project.org/web/packages/styler)
+  
+To install these packages, use the code below:
+
+```r
+# Required packages
+install.packages("dplyr")
+install.packages("ggplot2")
+install.packages("jsonlite")
+install.packages("R6")
+install.packages("tidyr")
+# Optional packages
+install.packages("crayon")
+install.packages("knitr")
+install.packages("parallel")
+install.packages("readxl")
+install.packages("Rmpi")
+install.packages("styler")
 ```
 
-or `Ctrl + Shift + L`
+## MS-Word reports
+
+It is possible to convert markdown reports to MS-Word (`.docx` format) from the `{ospsuite.reportingengine}` package.
+This conversion requires the installation of an additional software: [Pandoc](https://pandoc.org/).
+
+A dedicated article details how to create MS-Word reports using the `{ospsuite.reportingengine}` package: [here](https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/dev/articles/word-report.html)
+
+### [OPTIONAL] Pandoc Installation
+
+Install **Pandoc** (required for generation of reports in MS-Word format) by downloading one of the following files:
+
+- [Pandoc Installer (Windows)](https://github.com/jgm/pandoc/releases/download/2.9.2.1/pandoc-2.9.2.1-windows-x86_64.msi)
+- [Pandoc Installer (Linux)](https://github.com/jgm/pandoc/releases/download/2.9.2.1/pandoc-2.9.2.1-linux-amd64.tar.gz)

```

Comment:
let's link to a newer version 3.1.2
Theoretically we could link to the latest version of pandoc, but I didn't test it yet